lib/commandCenter/commandCenter.js

```diff
@@ -540,6 +540,23 @@ CommandCenter.prototype.executeCommand = function (cmd, session, metaData, cb) {
 		data: metaData
 	});
 
+	if (state.isBanned()) {
+		logger.info.data({
+			app: this.app.name,
+			module: cmdInfo.mod.name,
+			command: cmd.name,
+			user_acl: state.acl
+		});
+
+		return state.error('banned', null, function () {
+			// close the state and send the response
+
+			state.close(function (closeError, response) {
+				cb(null, response);
```

Comment:
Shouldn't the closeError be handled?

Reply:
I used the same code we use for acl state `auth` error so I'm not sure.
But we could modify to `cb(closeError, response)`.

Reply:
Its fine like that for now... let's see how it goes.